plugins/module_utils/falconctl_utils.py

```diff
@@ -50,7 +50,7 @@ def __get(opt):
     opt = opt.replace("_", "-")
     cmd.append("--%s" % opt)
     try:
-        stdout = check_output(cmd, universal_newlines=True, stderr=STDOUT)
+        stdout = check_output(cmd, universal_newlines=True, stderr=STDOUT, shell=False)
```

Comment:
@carlosmmatos will you check that this change doesn't affect things?

Reply:
This is fine